templates/settings/indexers.html

```diff
@@ -182,6 +182,24 @@ <h1>${_('Torrent Indexers')}</h1>
                 </div>
             </form>
 
+            <h1>Private Torrent Indexers</h1>
+            <form class="form-group row bg-light rounded mx-auto py-3" data-category="privtorrent">
+                <div class="input-group col-md-12">
+                    <div class="input-group-prepend">
+                        <span class="input-group-text">
+                            <i class="mdi mdi-checkbox-blank-outline c_box" data-id="danishbits" title="Enabled" value="${config['PrivateTorrent']['danishbits']['enabled']}"></i>
+                        </span>
+                    </div>
+                    <span class="input-group-item form-control">
+                        DanishBits
+                    </span>
+                </div>
+                <div class="input-group col-md-12">
+                    <input class="form-control" type="text" data-id="danishbits-username" placeholder="Username" value="${config['PrivateTorrent']['danishbits']['username']}"/>
+                    <input class="form-control" type="text" data-id="danishbits-passkey" placeholder="Passkey" value="${config['PrivateTorrent']['danishbits']['passkey']}"/>
+                </div>
+            </form>
+
```

Comment:
Does this look ok if everything is on one line? Use a `col-md-6 col-sm-12` so it doesn't get squished on small screens but also doesn't take up a ton of space on large screens.